src/coreclr/hosts/corerun/corerun.cpp

```diff
@@ -551,308 +329,298 @@ HRESULT InitializeHost(
     //
     // NATIVE_DLL_SEARCH_DIRECTORIES
     // - The list of paths that will be probed for native DLLs called by PInvoke
-    //
-    const char* property_keys[] = {
+    const char* propertyKeys[] =
+    {
         "TRUSTED_PLATFORM_ASSEMBLIES",
         "APP_PATHS",
         "APP_NI_PATHS",
```

Comment:
I think `APP_NI_PATHS` can be deleted. Nothing should depend on it anymore.

Reply:
You would think that, but unfortunately we do have tests that rely on it. That is something I was going to cleanup this is in.